internal/kube_helper.go

```diff
@@ -127,6 +127,14 @@ func NewConfigFromCommandline(kubeConfig string) (string, error) {
 	return "", nil
 }
 
+func GetKubeconfigPath() string {
```

Comment:
This func will return the value of `KUBECONFIG` env variable.
Can you please add a case to validate if the env variable value is empty (`""`), then the preflight and log-collector should error out before executing their respective checks. This scenario should respect the `--in-cluster` flag in case of preflight.